extensions/xdebug/lib/Executor/template/xdebug.template

```diff
@@ -0,0 +1,45 @@
+<?php
+
```

Comment:
add comment - this is the same as the standard executor template but doesn't return anything